loadflow/loadflow-api/src/main/java/com/powsybl/loadflow/LoadFlowProvider.java

```diff
@@ -43,15 +47,20 @@
      * to be stateless so that it can be call simultaneously with different arguments (a different network for instance)
      * without any concurrency issue.
      *
-     * @param network the network
+     * @param network            the network
      * @param computationManager a computation manager to external program execution
-     * @param workingVariantId variant id of the network
-     * @param parameters load flow execution parameters
-     * @param reporter the reporter used for functional logs
+     * @param workingVariantId   variant id of the network
+     * @param parameters         load flow execution parameters
+     * @param reporter           the reporter used for functional logs
      * @return a {@link CompletableFuture} on {@link LoadFlowResult]
      */
     default CompletableFuture<LoadFlowResult> run(Network network, ComputationManager computationManager, String workingVariantId, LoadFlowParameters parameters, Reporter reporter) {
         return run(network, computationManager, workingVariantId, parameters);
     }
 
+    // Defines the JSON serializer/deserializer to use
+    Optional<ExtensionJsonSerializer> getParametersExtensionSerializer();
+
+    // Load from platform config : will probably share code with the one above
+    <E extends Extension<LoadFlowParameters>> Optional<E> loadSpecificParameters(PlatformConfig config);
```

Comment:
I think we should be more consistent in the naming (sorry for the spec):
above we talk about "parameters extension", here about "specific parameters".

Maybe we can use the "specific parameters" naming in both places (the fact that it's an extension is more a technical detail):
`getSpecificParametersSerializer`
`loadSpecificParameters`